Sources/SwiftSyntax/Syntax.swift

```diff
@@ -380,6 +384,57 @@ extension Syntax {
   }
 }
 
+/// Temporary non-owning Syntax.
+///
+/// This can be used for handling Syntax node without ARC traffic.
+struct UnownedSyntax {
+  var raw: RawSyntax
+  var info: Unmanaged<Syntax.Info>
```

Comment:
Should these be `private` (or at least `info` be `private`). Seems like the intended access paths are via `value` and `parent`.